### DISCUSSION TITLE
Is there a way to disable snacks dashboard?

### DISCUSSION THREAD

**Discussion by MrRothstein** [LazyVim/LazyVim#4836] [Q&A] [answered]
I used to have this in my plugins:
```lua
{ "nvimdev/dashboard-nvim", enabled = false },
```

Now that dashboard comes from snacks, i'm not sure how to disable it... I tried:
```lua
return {
  "folke/snacks.nvim",
  opts = {
    dashboard = nil,
  },
}
```
along with some other variations, but nothing seems to prevent the dashboard.

**Comment by dpetka2001** [comment, marked as answer]
```lua
return {
  "folke/snacks.nvim",
  opts = {
    dashboard = { enabled = false },
  },
}
```
It's also mentioned in the [docs](https://github.com/folke/snacks.nvim?tab=readme-ov-file#%EF%B8%8F-configuration).
